package.json

```diff
@@ -33,6 +36,8 @@
     "deploy": "yarn run build && gh-pages --add --dist build/",
     "deploy-dev":
       "PUBLIC_URL=$npm_package_homepage_dev yarn run deploy --dest dev/",
+    "deploy-stage":
+      "POLLBOT_URL=$npm_package_pollbot_stage PUBLIC_URL=$npm_package_homepage_stage yarn run deploy --dest stage/",
```

Comment:
I still don't see the point of this. Why should DD Stage be paired with Pollbot Stage? If there's a strong reason, it should be spelled out in large letters in the README. Sorry for sounding negative but I just don't understand **why** this is. 

If I make a web app that interfaces with GitHub's API, my dev instance of that app uses api.github.com, not api-dev.github.com.

Reply:
Oh it is just a way to validate that everything works properly with the stage endpoint :dagger:

Reply:
There is no other point than using the pollbot stage endpoint with delivery-dashboard for all stacks.